workflows/single-file-artifact.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Single-file updates workflow",
+    "description": "Runs a single CLI command -- An invocation of the create_artifact CLI",
+    "version": 1,
+    "tasks": [
+        {
+            "name": "Run create_artifact CLI",
+            "type": "cli",
+            "cli": {
+                "command": [
+                    "create-artifact-worker",
```

Comment:
I see that the `CREATE_ARTIFACT_SERVER` env is required. should it just be an input parameter, and passed onto every single call on the CLI?
Like: `CREATE_ARTIFACT_SERVER=${workflow.input.artifact_server}` you reckon?

Reply:
@oleorhagen no no - these are env vars set per environment, at container startup.